compiler/hash-token/keywords/hash.gperf

```diff
@@ -0,0 +1,35 @@
+%{
```

Comment:
Could this file be generated by a macro similar to phf?

Reply:
Yeah, we could do it directly from rust, run the `gperf`, and output the `keywords.c` in the build file